View/SystemInfoViewBuilder.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzSupportToolsBundle\View;
+
+use Doctrine\Common\Inflector\Inflector;
+use eZ\Publish\Core\MVC\Symfony\View\Builder\ViewBuilder;
+use EzSystems\EzSupportToolsBundle\SystemInfo\Collector\SystemInfoCollector;
+
+class SystemInfoViewBuilder implements ViewBuilder
+{
+    /**
+     * @var \EzSystems\EzSupportToolsBundle\SystemInfo\Collector\SystemInfoCollector
+     */
+    private $infoCollector;
+
+    public function __construct(SystemInfoCollector $infoCollector)
+    {
+        $this->infoCollector = $infoCollector;
+    }
+
+    public function matches($argument)
+    {
+        return $argument === 'support_tools.view.controller:viewInfoAction';
+    }
+
+    public function buildView(array $parameters)
+    {
+        $view = new SystemInfoView();
+        $view->setInfo($this->infoCollector->build());
+        $view->setTemplateIdentifier(
+            $this->toTemplateIdentifier($view->getInfo())
```

Comment:
Note to self: must be changed.

The template identifier should come from the ViewMatchers (to @glye : executed by the Configurator, in the View Request Listener).

It could be interesting to base the default template name on a convention, probably from the view builder (computed based on the class name, as it is now), with a convention based directory (`HardwareSystemInfo` => `ez-support-tools/hardware_system_info.html.twig`).

We are kind of getting close from the concept we had in legacy, where the _requested template_ was part of the view rule. We don't have this feature yet (matchers can't/don't test that).

Shorter version: if a default template is set by convention, based on the rendered info, in the view object, how could we take advantage of that in view rules ?